gzip_test.go

```diff
@@ -208,6 +208,51 @@ func TestGzipHandlerNoBody(t *testing.T) {
 	}
 }
 
+func TestGzipHnalderStream(t *testing.T) {
```

Comment:
did you mean write `TestGzipHandlerStream` instead of `TestGzipHnalderStream`

Reply:
Woops yep! Also I’m not confident in the fix but the test case reproduces the issue at hand. 

Startgzip gets called but not init then subsequently startPlain gets called and then errors